ci-operator/step-registry/ipi/conf/azure/ipi-conf-azure-commands.sh

```diff
@@ -50,24 +50,25 @@ workers=${COMPUTE_NODE_REPLICAS:-3}
 if [ "${COMPUTE_NODE_REPLICAS}" -le 0 ] || [ "${SIZE_VARIANT}" = "compact" ]; then
   workers=0
 fi
-master_type=null
-master_type_prefix=""
+master_type_prefix="Standard_D4s_v3"
 if [[ "${SIZE_VARIANT}" == "xlarge" ]]; then
   master_type_prefix=Standard_D32
 elif [[ "${SIZE_VARIANT}" == "large" ]]; then
   master_type_prefix=Standard_D16
 elif [[ "${SIZE_VARIANT}" == "compact" ]]; then
   master_type_prefix=Standard_D8
 fi
-if [ -n "${master_type_prefix}" ]; then
-  if [ "${OCP_ARCH}" = "amd64" ]; then
-    master_type=${master_type_prefix}s_v3
-  elif [ "${OCP_ARCH}" = "arm64" ]; then
-    master_type=${master_type_prefix}ps_v5
+if [[ "${CONTROL_PLANE_INSTANCE_TYPE}" == "" ]]; then
+  if [ -n "${master_type_prefix}" ]; then
+    if [ "${OCP_ARCH}" = "amd64" ]; then
+      CONTROL_PLANE_INSTANCE_TYPE=${master_type_prefix}s_v3
+    elif [ "${OCP_ARCH}" = "arm64" ]; then
+      CONTROL_PLANE_INSTANCE_TYPE=${master_type_prefix}ps_v5
+    fi
```

Comment:
The original codes are already well tested, to be safe, I suggest not change the original code, just add the following code to override.
```
if [[ -n "${CONTROL_PLANE_INSTANCE_TYPE}" ]]; then
    master_type="${CONTROL_PLANE_INSTANCE_TYPE}"
fi
```